cebra/attribution/attribution_models.py

```diff
@@ -0,0 +1,591 @@
+#
+# CEBRA: Consistent EmBeddings of high-dimensional Recordings using Auxiliary variables
+# © Mackenzie W. Mathis & Steffen Schneider (v0.4.0+)
+# Source code:
+# https://github.com/AdaptiveMotorControlLab/CEBRA
+#
+# Please see LICENSE.md for the full license document:
+# https://github.com/AdaptiveMotorControlLab/CEBRA/blob/main/LICENSE.md
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import dataclasses
+import time
+
+import cvxpy as cp
+import numpy as np
+import scipy.linalg
+import sklearn.metrics
+import torch
+import torch.nn as nn
+import tqdm
+from captum.attr import NeuronFeatureAblation
```

Comment:
Do we really need all the baselines in this package? We should write up docs for this also ...

Reply:
we can leave them, but we need to then add docs